webapp/src/components/Sidebar.js

```diff
@@ -250,14 +268,16 @@ SidebarCategory.propTypes = {
 }
 
 const SidebarLink = ({ name, icon, to, badge }) => {
+  console.log('=>', to)

```

Comment:
do we need the console log?

Reply:
My mistake I forget remove it, it's fixed in my last commit